bluepyopt/ephys/simulators.py

```diff
@@ -89,6 +89,12 @@ def neuron(self):
             NrnSimulator._nrn_disable_banner()
             self.banner_disabled = True
 
+        # certain mpi libraries use PMI_RANK env variable to initialize
+        # MPI before calling MPI_Init (which is undesirable). Unset this
+        # variable to avoid issue with loading neuron and mechanism library.
+        if 'PMI_RANK' in os.environ:
+            os.environ.pop("PMI_RANK")
```

Comment:
```suggestion
            del os.environ["PMI_RANK"]
```
? Seems a little more pythonic…

Reply:
True. On [this](https://stackoverflow.com/questions/18178038/how-to-clear-os-environ-value-for-only-one-variable-in-python/18178048) Stackoverflow question I saw people voted more for `pop` and hence used it.